source/help/apps/desktop-changelog.md

```diff
@@ -0,0 +1,333 @@
+# Desktop Application Changelog
+
+## Release v3.4.0
+
+Expected release date: End of September 2016
+
+This release contains a security update and it is highly recommended that users upgrade to this version.
+
+Version number updated to 3.4 to make numbering consistent with Mattermost server and mobile app releases. This change will not imply monthly releases.
+
+### Improvements
+
+#### Windows
+ - Current team and channel name shown in window caption
```

Comment:
The term "window caption" seems a little Expert Mode, is there a more simple way to explain this?